index.bs

```diff
@@ -5294,6 +5295,13 @@ partial interface MLGraphBuilder {
     1. If [=MLGraphBuilder/validating operand=] with [=this=] and |input| returns false, then [=exception/throw=] a {{TypeError}}.
     1. If any of |sizes|'s [=list/items=] are 0, then [=exception/throw=] a {{TypeError}}.
     1. If |starts|'s [=list/size=] and |sizes|'s [=list/size=] are not both equal to |input|'s [=MLOperand/rank=], then [=exception/throw=] a {{TypeError}}.
+    1. [=list/For each=] |index| in [=the range=] 0 to |input|'s [=MLOperand/rank=], exclusive:
+        1. If |sizes|[|index|] is 0, then [=exception/throw=] a {{TypeError}}.
+
+            Issue(391): If 0-size dimensions are allowed, revise these steps.
+
+        1. If |starts|[|index|] is greater than or equal to |input|'s [=MLOperand/shape=][|index|], then [=exception/throw=] a {{TypeError}}.
```

Comment:
If the 0-size dimensions are allowed, would it be allowed that the |starts|[|index|] is equal to |input|'s [=MLOperand/shape=][|index|]? Or is the next validation step good enough?

Reply:
I _think_ we'd want to allow `starts[index]` equal to `input's shape[index]` so the resulting slice is 0, so yeah this step would need to be revised too. The note is written to be pretty generic and mentioned "these steps" - were you wanting something more here?

Reply:
> The note is written to be pretty generic and mentioned "these steps"

Makes sense, it is good for now.